plugins/jobs/src/js/pages/JobTaskDetailPage.js

```diff
@@ -30,6 +59,16 @@ class JobTaskDetailPage extends React.Component {
     ];
 
     const task = MesosStateStore.getTaskFromTaskID(taskID);
+
+    if (!task) {
+      return (
+        <Trans>
+          Either the data related to that task has already been cleaned up or
```

Comment:
I think "this" instead of "that" sounds better.

Reply:
totally agree!